src/mopi/plugins/utils.py

```diff
@@ -91,7 +91,7 @@ def print_example_outputs(
         [], columns=["input text", "final_output", "original_labels", "confidence"]
     )
 
-    random_indecies = random.sample(range(len(input)), num_examples)
+    random_indecies = random.sample(range(len(input)), min(len(input),num_examples))
```

Comment:
This was getting a random number of elements from the input source, but given that it might be smaller I fixed it.